assets/src/edit-story/components/inspector/prepublish/test/prepublishChecklistProvider.js

```diff
@@ -92,15 +92,14 @@ function setup({ pageCount = 1 }) {
 }
 
 describe('prepublishChecklistProvider', () => {
-  it(`should begin a story at ${PPC_CHECKPOINT_STATE.UNAVAILABLE}`, async () => {
+  // TODO this should be set to UNAVAILABLE once blank story trigger is ready
+  it(`should begin a story at ${PPC_CHECKPOINT_STATE.ALL}`, async () => {
```

Comment:
Sometimes I find it helpful to wrap code/states in some quotes to make it more readable in a sentence
```suggestion
  it(`should begin a story at '${PPC_CHECKPOINT_STATE.ALL}'`, async () => {
```